pkg/connector/event_log.go

```diff
@@ -0,0 +1,200 @@
+package connector
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/conductorone/baton-jira/pkg/client"
+	v2 "github.com/conductorone/baton-sdk/pb/c1/connector/v2"
+	"github.com/conductorone/baton-sdk/pkg/annotations"
+	"github.com/conductorone/baton-sdk/pkg/pagination"
+	"github.com/grpc-ecosystem/go-grpc-middleware/logging/zap/ctxzap"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+const (
+	defaultPageSize = 100
+)
+
+// auditFilters defines the Jira audit log event types we're interested in.
+var auditFilters = []string{
+	"Deleted Jira issue",
+	"Field added to Screen",
+	"Field removed from Screen",
+	"Issue type updated",
+	"Workflow updated",
+	"Project updated",
+	"Previous project keys released",
+	"Project avatar changed",
```

Comment:
This feels like it would be a very rare event. Why would we be interested in this event?